src/core/net/ip6_headers.hpp

```diff
@@ -106,14 +106,14 @@ enum IpProto
  */
 enum IpDscpCs
 {
-    kDscpCs0    = 0,    ///< Class selector codepoint 0
-    kDscpCs1    = 8,    ///< Class selector codepoint 8
-    kDscpCs2    = 16,   ///< Class selector codepoint 16
-    kDscpCs3    = 24,   ///< Class selector codepoint 24
-    kDscpCs4    = 32,   ///< Class selector codepoint 32
-    kDscpCs5    = 40,   ///< Class selector codepoint 40
-    kDscpCs6    = 48,   ///< Class selector codepoint 48
-    kDscpCs7    = 56,   ///< Class selector codepoint 56
+    kDscpCs0    = 0,    ///< Class selector code point 0
```

Comment:
IANA spells "codepoint" as one word - see [DSCP Registry](https://www.iana.org/assignments/dscp-registry/dscp-registry.xhtml).

Reply:
Thanks. Changed it back to "codepoint".

I checked wiki page (https://en.wikipedia.org/wiki/Differentiated_services) and noticed it was using "code point"....should not trust wiki... :)